static/sass/_layout_documentation.scss

```diff
@@ -0,0 +1,122 @@
+@mixin layout-documentation {
+  @include pattern-sidebar-nav;
+
+  .l-documentation {
+    @include row;
+    display: flex;
+
+    .p-notification,
+    .p-notification--caution,
+    .p-notification--floating .p-notification,
+    .p-notification--information,
+    .p-notification--negative,
+    .p-notification--positive {
+      margin: 2rem 0.1rem;
+    }
+
+    pre {
+      max-width: 700px;
+      overflow-x: auto;
+      width: 95vmax;
+    }
+
+    pre,
+    code {
+      white-space: pre;
+    }
+
+    img {
+      margin: 2rem 0 2rem 0;
+    }
+
+    h2,
+    h3,
+    h4 {
+      padding-top: 1.55rem;
+    }
+
+    @media (max-width: $breakpoint-medium) {
+      flex-direction: column;
+    }
+
+    &__sidebar {
+      border-right: 1px solid $color-mid-light;
+      flex: 0 0 18rem;
+
+      @media (max-width: $breakpoint-medium) {
+        border-bottom: 1px solid $color-mid-light;
+        border-right: 0;
+        flex: 1;
+        flex-direction: column;
+      }
+    }
+
+    &__content {
+      flex: 5;
+      margin: 2rem 0;
+      padding-left: 5%;
```

Comment:
Can you make this only large screens please

Reply:
And the margin

Reply:
done